### PR TITLE
Develop with okteto

### PR DESCRIPTION
**Pull request by AgustinRamiroDiaz**
Signed-off-by: Agustin Ramiro Diaz <agustin.ramiro.diaz@gmail.com>

With this, you can run directly `okteto up` and develop the docs with okteto

### CODE REVIEW

**Review by bgonp — APPROVED** [review]

**Comment by AgustinRamiroDiaz** [comment]
Maybe we should wait until [this](https://github.com/okteto/okteto/issues/2609) is fixed to merge

File: README.md [hunk 1]
```diff
@@ -4,6 +4,25 @@
 
 This repository is the source for https://okteto.com/docs. If you'd like to contribute to the documentation, please first check out our [contributing guide](CONTRIBUTING.md) for info on how to get started.
 
+## Development
+
+### ☁️ Develop with okteto
+
+- [Install okteto cli](https://www.okteto.com/docs/getting-started/#installing-okteto-cli)
```
Comment (RinkiyaKeDad) [inline README.md:11]:
```suggestion
- [Install Okteto CLI](https://www.okteto.com/docs/getting-started/#installing-okteto-cli)
```

File: README.md [hunk 2]
```diff
@@ -4,6 +4,25 @@
...
+- [Configure okteto context for cloud](https://www.okteto.com/docs/getting-started/#configuring-okteto-cli-with-okteto-cloud)
```
Comment (RinkiyaKeDad) [inline README.md:12]:
```suggestion
- [Set your `okteto context`](https://www.okteto.com/docs/getting-started/#configuring-okteto-cli-with-okteto-cloud)
```

File: README.md [hunk 3]
```diff
@@ -4,6 +4,25 @@
...
+- Clone the repo and then run `okteto up` from the terminal
```
Comment (RinkiyaKeDad) [inline README.md:13]:
```suggestion
- Clone the repo and run `okteto up` from the root of the repo
```

**Review by RinkiyaKeDad — APPROVED** [review]

**Pull request merged**